apps/prairielearn/elements/pl-matrix-component-input/pl-matrix-component-input.py

```diff
@@ -372,14 +383,21 @@ def grade(element_html: str, data: pl.QuestionData) -> None:
     weight = pl.get_integer_attrib(element, "weight", WEIGHT_DEFAULT)
 
     # Get method of comparison, with relabs as default
-    comparison = pl.get_string_attrib(element, "comparison", COMPARISON_DEFAULT)
+    raw_comparison = pl.get_string_attrib(element, "comparison", COMPARISON_DEFAULT)
```

Comment:
Probably better off changing this to an enum and using `get_enum_attrib`.

Reply:
Fixed in [3cd2306](https://github.com/PrairieLearn/PrairieLearn/pull/11188/commits/3cd23066a1f97e5fc88538e0cdc1a4511419e5ea).